modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -207,6 +208,27 @@ const addButtons = (editor: Editor, selectionTargets: SelectionTargets, clipboar
       onSetup: selectionTargets.onSetupCellOrRow
     });
   }
+
+  editor.ui.registry.addMenuButton('tablecellvalign', {
```

Comment:
Something that I missing in your last PR, but for TinyMCE we have a general rule that if you have a toolbar button, there should be an equivalent menu item (or vice-versa). So we'll also need menu items registered for this.

Note: I'll add this task to the previous PR as well sorry.